src/env/development.js

```diff
@@ -41,7 +41,7 @@ import { Pairwise } from "../examples/pairwise"; // eslint-disable-line no-unuse
  */
 // import { AllTypes } from "../examples/all_types"; // eslint-disable-line no-unused-vars
 
-const data = ImageBbox;
+const data = AudioRegions;
```

Comment:
This line is constantly gets changed and committed. We need to think of how that can be configured and edited locally.